Netflix/Views/Dashboard/ComingSoon/CommingSoonUIView.swift

```diff
@@ -77,13 +77,13 @@ class CommingSoonUIView: UIView, UICollectionViewDelegate, UICollectionViewDataS
 }
 
 // Set settings functions
-extension CommingSoonUIView {
+extension ComingSoonUIView {
```

Comment:
If it's UICollectionViewDelegate, I would move conformance here
```suggestion
extension ComingSoonUIView: UICollectionViewDelegate {
```